Gopkg.toml

```diff
@@ -69,15 +56,4 @@ required = [
 [prune]
   go-tests = true
   non-go = true
-
-  [[prune.project]]
-    name = "k8s.io/code-generator"
-    non-go = false
-
-  [[prune.project]]
-    name = "k8s.io/gengo"
-    non-go = false
-  
-  [[prune.project]]
-    name = "github.com/enmasseproject/enmasse"
-    unused-packages = true
+  unused-packages = true
```

Comment:
This reduces the number of files that we have in `vendor/`, but it means that we have to run `dep ensure` again if we want to use any package from a current dependency that we're not currently using.